pkgs/development/compilers/qbe/default.nix

```diff
@@ -16,6 +16,13 @@ stdenv.mkDerivation (finalAttrs: {
 
   doCheck = true;
 
+  enableParallelBuilding = true;
+
+  patches = [
+    # Use "${TMPDIR:-/tmp}" instead of the latter directly.
```

Comment:
Could you add a link to the upstream issue/thread about this?

Reply:
Added the link.